### DISCUSSION TITLE
[Unstable] Bandit beacons don't have reactor, supercapacitor turned on

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14007] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bandit reactor isnt turned on by event = no power

Add this to `beaconbandit_enablelayers` event

```xml
      <StatusEffectAction targettag="beaconreactor" >
        <StatusEffect target="This" PowerOn="true" />
      </StatusEffectAction>
```

### Reproduction steps

1. leveleditor
2. spawn level with Beaconstation_Jagala
3. `triggerevent beaconbandits5`
4. no power

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
> Bandit reactor isnt turned on by event = no power

The bandits have the operatereactor order though, they should be able to turn it on themselves. I think that change should already be on unstable, did the bots not operate the reactor in your tests?

**Reply by ubertpendragon** [reply]
The bots did not operate the reactor and as a result they run out of power for turrets

**Reply by JeroonBosch** [reply]
If that's the case we should fix the bots not operating the reactor instead, creating a ticket for that.

**Comment by NilanthAnimosus** [comment]
Closing as fixed and merged. Should be in a future build (The AI bandits spawning in beacons via the events now operate the reactors correctly).
